package.json

```
@@ -11,12 +11,18 @@
     "dev": "ts-node-dev --files --transpile-only --respawn --no-notify ./server.ts | pino-pretty",
     "build": "npm run clean && tsc -b",
     "clean": "tsc -b --clean",
-    "test": "TS_NODE_FILES=true tap --ts --no-check-coverage --test-ignore='^(dist|scripts)'",
+    "test": "TS_NODE_FILES=true NODE_OPTIONS=\"-r ts-node/register\" c8 ts-node --transpile-only node_modules/tape/bin/tape **/*.test.ts | tap-arc",
```

```suggestion
    "test": "TS_NODE_FILES=true NODE_OPTIONS=\"-r ts-node/register\" c8 ts-node --transpile-only node_modules/.bin/tape **/*.test.ts | tap-arc",
```

Very slightly more reliable (in case tape changes the location of the bin script in their repo)